src/Appium.Net/Appium/AppiumOptions.cs

```diff
@@ -8,38 +11,160 @@ namespace OpenQA.Selenium.Appium
     /// </summary>
     public class AppiumOptions : DriverOptions
     {
+        private const string AutomationNameOption = "appium:automationName";
+        private const string DeviceNameOption = "appium:deviceName";
+        private const string AppOption = "appium:app";
+        private const string PlatformVersionOption = "appium:platformVersion";
+        
+        private readonly Dictionary<string, object> additionalAppiumOptions = new Dictionary<string, object>();
+
+
         /// <summary>
-        /// The dictionary of capabilities
+        /// Initializes a new instance of the <see cref="AppiumOptions"/> class.
         /// </summary>
-        private readonly AppiumCapabilities capabilities = new AppiumCapabilities();
+        public AppiumOptions() : base()
+        {
+            this.AddKnownCapabilityName(AppiumOptions.AutomationNameOption, "AutomationName property");
+            this.AddKnownCapabilityName(AppiumOptions.DeviceNameOption, "DeviceName property");
+            this.AddKnownCapabilityName(AppiumOptions.AppOption, "Application property");
+            this.AddKnownCapabilityName("app", "Application property");
```

Comment:
why should we add it without the prefix? It is against the spec

Reply:
A bit torn on if we should have it or not.
If a user does something like this:
`appCapabilities.AddAdditionalOption("app", "Microsoft.WindowsCalculator_8wekyb3d8bbwe!App");`
*This currently just works with the latest Appium.net release
The setup will fail and tell them they never defined and application.
This way you get a useful error message.

@akinsolb - Would love to get your 2 cents here

Reply:
@TroyWalshProf only just seeing this. Think it's best to add the prefix. The driver options can override it.